### ISSUE TITLE
Extending the Web Blog [Feature request]

### ISSUE THREAD

**Issue by ClausSamuelsen** [gramps-project/gramps-web#443]
I really like the Blog feature in the Gramps Web, but I would like it to be able to be more "blog" like, e.g. having images between text blocks as well as sub headers etc.
My immediate idea is to have multiple notes for one source and to use markdown in the notes.

**Comment by DavidMStraub** [comment]
Agreed that having images in between text would be very useful. One guiding principle has to be that the note format is compatible with Gramps desktop though, so allowing Markdown is not an option until it's supported by Gramps core.

**Comment by ClausSamuelsen** [comment]
I don't see how markdown characters (##, ** etc) in a note can conflict with Gramps desktop, as markdown is just text. Gramps desktop doesn't need to do anything with markdown in notes, same as Gramps desktop doesn't need to do anything about a source that has a tag 'Blog' attached.
Besides that, I fully agree that Gramps Web features must not conflict with Gramps desktop.

**Comment by DavidMStraub** [comment]
> I don't see how markdown characters (##, ** etc) in a note can conflict with Gramps desktop, as markdown is just text. Gramps desktop doesn't need to do anything with markdown in notes, same as Gramps desktop doesn't need to do anything about a source that has a tag 'Blog' attached.

I don't agree, mixing Markdown syntax and Gramps styled text will lead to all kind of annoying behaviour. The only thing I could imagine is using Markdown *while typing*, and converting it to the appropriate tags when storing the note. But Gramps styled text doesn't support headings and many other Markdown features. So let's ignore Markdown & subheadings for now and focus on the images between text in this issue.

**Comment by DavidMStraub** [comment]
On the images between text blocks: I thought about it and I think the best solution would be to use

> Text 1
> 
> [Link to image](#)
> 
> Text 2

The "Link to image" would be just a link in Gramps Desktop, and would turn into the caption of the image on Gramps Web.

**Comment by nielstron** [comment]
I think in general it would be nice to have a small helper for font formatting (like on GitHub) + Preview.
like this.

<img width="1232" height="562" alt="Image" src="https://github.com/user-attachments/assets/5867069a-5f0c-4e5e-8350-88a7459dd66d" />
